### PR TITLE
[tmcf + csv] Load tmcf and csv from GCS; Update server flags

### PR DESCRIPTION
**Pull request by shifucun**
- Now have a flag for each storage: bigquery, base bigtable, branch bigtable and gcs.
- Read tmcf and csv from GCS, parse tmcf into table schema and read in csv rows.
- Next step is to construct time series from each csv row based on table schema.

### CODE REVIEW

File: internal/store/memdb.go [hunk 1]
```diff
@@ -15,30 +15,110 @@
...
+			if _, err := io.Copy(buf, r); err != nil {
+				return err
+			}
+			schemaMapping, err = parser.ParseTmcf(buf.String())
+			if err != nil {
+				return err
+			}
+			break
+		}
+	}
```
Comment (pradh) [inline internal/store/memdb.go:83]: If you have multiple TMCFs and CSVs, how do we match them up?

**Review by pradh — APPROVED** [review]

File: internal/store/memdb.go (hunk 1 above)
Comment (pradh) [reply internal/store/memdb.go:83]: One option is we could have have a directory object per import, where each directory object has one TMCF and one or more CSVs.

Comment (shifucun) [reply internal/store/memdb.go:83]: The argument "gcs_object_prefix" is used to hold one import. I make it more clear in the description.

Comment (pradh) [reply internal/store/memdb.go:86]: SG, then

**Review by pradh — APPROVED** [review]

**Pull request merged**